nixos/modules/services/web-apps/cryptpad.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cryptpad;
+in
+{
+  options.services.cryptpad = {
+    enable = mkEnableOption "the Cryptpad service";
+
+    package = mkOption {
+      default = pkgs.cryptpad;
```

Comment:
I personally feel like we shouldn't add `package` options when there's no need to, but [not everyone feels the same](https://github.com/NixOS/nixpkgs/issues/50476).

Reply:
Ok let me contribute to that other thread then.

Reply:
Looks like I already commented: https://github.com/NixOS/nixpkgs/issues/50476#issuecomment-439534060